schedule-tracking/src/main/java/org/motechproject/scheduletracking/repository/AllEnrollments.java

```diff
@@ -10,36 +10,82 @@
 import java.util.HashSet;
 import java.util.List;
 
+/**
+ * Implementation of DAO interface that utilizes a MDS back-end for retrieving instances.
```

Comment:
This does not implement any interfaces. Just say that this is a DAO that utilizes..